packages/ui/src/primitives/icon.tsx

```diff
@@ -249,6 +249,17 @@ export const Linear = createIcon({
   ),
 });
 
+export const LinkedIn = createIcon({
```

Comment:
Maybe we remove `LinkedInOIDC` and just use `LinkedIn` since they are both the same icon?

Reply:
Yeah that makes sense to me; I only did it initially because I was using the icon name as a key